### PR TITLE
Maintenance

### PR DESCRIPTION
**Pull request by mdeff**
Some maintenance on the infrastructure. Should be kept in sync with https://github.com/epfl-lts2/pygsp/pull/81 to facilitate future maintenance by keeping their infra similar.

### CODE REVIEW

**Comment by nperraud** [comment]
What do you want to still do here?
I think merging very soon is a goo idea so we can have working tests.

**Comment by mdeff** [comment]
Some uncomitted changes that I still have. Most notably evaluating a transition from [numpydoc](https://numpydoc.readthedocs.io) to [napoleon](https://www.sphinx-doc.org/en/master/usage/extensions/napoleon.html). I'm pushing some now.

But the tests work on the pyunlocbox without this PR. On the PyGSP it doesn't fix the tests yet because graph-tool doesn't provide a package for Ubuntu Xenial anymore.

**Comment by nperraud** [comment]
For graph tool I think you are wrong. See the following link:

https://downloads.skewed.de/graph-tool/packages/

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 language: python
 
 python:
-  - 2.7
-  - 3.5
   - 3.6
   - 3.7
+  - 3.8
+  - 3.9
```
Comment (nperraud) [inline .travis.yml:7]: I added python 3.9

**Review by nperraud — APPROVED** [review]
Everything looks fine. Let us merge.

**Comment by mdeff** [comment]
Finally 🎉

**Comment by nperraud** [comment]
youhou, thanks

**Pull request merged**